bin/nyc.js

```diff
@@ -16,23 +16,47 @@ var yargs = require('yargs/yargs')(process.argv.slice(2))
   .command('report', 'run coverage report for .nyc_output', function (yargs) {
     return yargs
       .usage('$0 report [options]')
+      .option('reporter', {
+        alias: 'r',
+        describe: 'coverage reporter(s) to use',
+        default: 'text'
+      })
+      .option('report-dir', {
+        describe: 'default directory to output coverage reports in',
```

Comment:
Shouldn't this simply say:

```
directory to output coverage reports in
```

The default is `coverage`. If `report-dir` is specified, it's no longer the "default".